docs/articles/documentation/using-testcafe/command-line-interface.md

```diff
@@ -213,6 +213,14 @@ TestCafe will run fixtures whose names match the specified pattern.
 testcafe ie my-tests --fixture-grep *Page*
 ```
 
+### --element-timeout \<ms\>
+
+Specifies the amount of time, in milliseconds, allowed for a page element to become available before the test fails.
```

Comment:
Or, even better we should define what `available` means (becomes visible and present in the DOM).